src/urllib3/util/request.py

```diff
@@ -66,7 +67,7 @@ def make_headers(
         >>> make_headers(accept_encoding=True)
         {'accept-encoding': 'gzip,deflate'}
     """
-    headers = {}
+    headers: Dict[str, Any] = {}
```

Comment:
Why is the value type `Any` instead of `str`?

Reply:
I was wrong. `str` is the right type.